src/modules/olcs/Service.js

```diff
@@ -15,6 +48,16 @@ const Service = class {
    */
   initialize(manager) {
     this.manager_ = manager;
+
+    this.manager_.getLoadPromise().then(() => {
+      this.setupPermalink_();
+    });
+
+    if (this.ngeoStateManager_.getInitialBooleanValue('3d_enabled')) {
+      this.manager_.toggle3d().then(() => {
```

Comment:
It looks strange to toggle 3d and only then set the correct position. With the above code:
- the globe will tilt with an animation to 3D at some default position and angle;
- then it will change (jump?) to the position/angle from the permalink.

Reply:
@gberaudo if i don't do this, then the 3d view is mapped with the 2d one, and not with the peralink parameters.